src/python/WMCore/MicroService/MSOutput/MSOutput.py

```diff
@@ -248,6 +249,53 @@ def _executeConsumer(self, summary):
             self.logger.exception(msg)
             self.updateReportDict(summary, "error", msg)
 
+    def setDBSStatus(self, workflow):
+        """
+        The function to set the DBS status of outputs as VALID
+        :param workflow: a MSOutputTemplate object workflow
+        :return: the MSOutputTemplate object itself (with the necessary updates in place)
+        """
+        if not isinstance(workflow, MSOutputTemplate):
+            msg = "Unsupported type object '{}' for workflows! ".format(type(workflow))
+            msg += "It needs to be of type: MSOutputTemplate"
+            raise UnsupportedError(msg)
+
+        dbs3Writer = DBS3Writer(url=self.msConfig["dbsReadUrl"],
```

Comment:
I would suggest to initialize it in the `__init__` method instead.

Reply:
Should be fine now